### PR TITLE
Add/remove tasks from to-do list

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
- This project used WebPack modules to implement a fully functional To-Do List where the user can add, remove or edit their tasks from the list. The added and removed tasks also reflect in the local storage

**Highlights**

- Setup project with WebPack
- Use JavaScript to dynamically add the items the user adds to the list.
- Remove the items on clicking the delete button
- Update local storage with the changes made
- Set up linters and fix linter errors

**Cheers and Happy Reviewing**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
+  e.preventDefault();
+  Events.addList();
+});
+deleteButton.addEventListener('click', (e) => {
+  const clicked = e.target.closest('.fa-trash-can');
+  if (!clicked) return;
+  const listIndex = +clicked.dataset.index;
+  Store.removeTasks(listIndex);
+  Events.displayTasks();
+  location.reload();
```
Comment (rustam-tolipov) [inline src/index.js]: In the requirement: 

> Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

However, when I try to delete one of the tasks it's not updating the list in the local store as you can see in this image:

Screen Shot 2022-07-16 at 4 53 27 AM

- [ ] Kindly solve this issue.

File: src/modules/store.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable max-classes-per-file */
+// local Storage
+export class Task {
```
Comment (rustam-tolipov) [inline src/modules/store.js:3]: - [ ] Also, the edit task is not implemented. It's not hard to give suggestions if you can't describe where are you stuck. Please write which point of creating an edit task is stopping you.

File: README.md [hunk 3]
```diff
@@ -9,15 +9,30 @@
...
+### steps to run the project
+#### 1 - Install npm.
+```
+npm install
+```
+
+#### 2 - run the project 
+```
+npm run dev
+```
```
Comment (rustam-tolipov) [inline README.md:35]: - [ ] [OPTIONAL] Kindly implement the MIT license to you project as well.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @sharon-odhiambo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/store.js (hunk 2 above)
Comment (sharon-odhiambo) [reply src/modules/store.js:3]: Hi @Rustamxon7 , thank you for your review. 
In my ui.js file from line 39 to 45, I tried to implement the editing function but it's not working properly. Could you kindly check and help me know where am going wrong?

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @sharon-odhiambo  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — COMMENTED** [review]
Hi @sharon-odhiambo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has already been approved

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/store.js (hunk 2 above)
Comment (rustam-tolipov) [reply src/modules/store.js:3]: Hello @sharon-odhiambo, Sorry I was not able to reply in time to you. But looks like you solve the issue, right?

Comment (sharon-odhiambo) [reply src/modules/store.js:3]: Hi, yeah sure. I did. Thank you.
No worries at all.

**Pull request merged**